package.json

```
@@ -84,7 +84,7 @@
     "prettier-plugin-svelte": "2.7.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-query": "3.39.1",
+    "react-query": "4.0.0",
```

```suggestion
    "@tanstack/react-query": "4.0.3",
```
https://tanstack.com/query/v4/docs/installation